courses/cunix/ex01/src/my_strlen.c

```diff
@@ -0,0 +1,9 @@
+unsigned int my_strlen(char *str){
+	unsigned int num=0;
+	//char ch;
+	while(*str!='\0'){
```

Comment:
also you should put a space after while, for etc
this makes code more readable and is considered as clean code style